sds_data_manager/stacks/dynamodb_stack.py

```diff
@@ -0,0 +1,139 @@
+# Create stack for dynamoDb
+from aws_cdk import RemovalPolicy, Stack
+from aws_cdk import aws_dynamodb as dynamodb
+from constructs import Construct
+
+
+class DynamoDB(Stack):
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        sds_id: str,
+        table_name: str,
+        partition_key: str,
+        sort_key: str,
+        env,
+        on_demand=True,
+        read_capacity=1,
+        write_capacity=1,
```

Comment:
```suggestion
        env: Environment,
        on_demand: bool = True,
        read_capacity: int = 1,
        write_capacity:int = 1,
```
I'd suggest to keep the typing going on all of the arguments.